static/styles/settings.css

```diff
@@ -386,6 +386,16 @@ td .button {
 .button,
 .input-group {
     margin: 0 0 20px;
+
+    &#user_deactivate_account_button {
```

Comment:
This CSS can be defined at the top level, since you are using ID only.

Reply:
have made the changes.